### ISSUE TITLE
Is there a way to start the navigation directly in the background?

### ISSUE THREAD

**Issue by awaisabbasaa11** [mapbox/mapbox-maps-android#405]
I have a unique requirement where I want to start the navigation in the background and in the foreground I want to open and interact with an other activity (Camera Activity for Object detection).

What I mean by background is when I use NavigationLauncher like this
NavigationLauncher.startNavigation(MapRouteActivity.this, options);

I want it to start the navigation in the background (show notification in the notification tray) and guide the user with the voice alerts meanwhile some other activity is in the foreground doing its work.

Any help/idea/suggestion will be highly appreciated 

Thanks

**Comment by knov** [comment]
@ awaisabbasaa11 are you using the Mapbox Navigation SDK's [NavigationLauncher](https://docs.mapbox.com/android/navigation/api/ui/0.42.6/com/mapbox/services/android/navigation/ui/v5/NavigationLauncher.html) API? 

cc @LukasPaczos @zugaldia @truburt

**Comment by awaisabbasaa11** [comment]
Yes, and I have achieved this. I noticed that you already provide navigation in the background and a notification in the notification tray that also provide Voice alerts, so to implement my requirement I had to do something like this

Launching the navigation activity (By doing this: NavigationLauncher.startNavigation(MapRouteActivity.this, options);)
then after some delay of millisecond launched the second activity (Object Detection Activity) in a new Task.
